### PR TITLE
Add a mono-mixin and overhaul the public API

### PR DESCRIPTION
**Pull request by rowanmanning**
This adds a mono-mixin and renames some of the other mixins to refine
the public API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 Tabs component for dividing content into meaningful sections.
 
 - [Usage](#usage)
```
Comment (notlee) [inline README.md]: We've been removing the top level "Usage" heading

File: README.md [hunk 2]
```diff
@@ -144,29 +144,78 @@ To apply the __buttontabs__ styling, add a `o-tabs--buttontabs` class to the roo
...
-or use the mixin:
+The __buttontabs__ style comes in two sizes that conform to the `o-buttons` sizes: medium and big. Medium is the default size and big can be applied by adding the class `o-tabs--big`.
+
+#### Options
+
+* __Align right__: Add `o-tabs--alignright` to the root element.
+
+#### Silent mode
+
+As with all Origami components, o-tabs has a [silent mode](https://origami.ft.com/spec/v1/sass/#sass-silent-mode). To use its compiled CSS (rather than incorporating its mixins into your own Sass) set `$o-tabs-is-silent: false;` in your Sass before you import the o-tabs Sass:
```
Comment (notlee) [inline README.md]: We've been deleting silent mode documentation from READMEs in favour of the mono mixin. E.g.
https://github.com/Financial-Times/o-table#sass
https://github.com/Financial-Times/o-forms#sass

File: README.md [hunk 3]
```diff
@@ -144,29 +144,78 @@ To apply the __buttontabs__ styling, add a `o-tabs--buttontabs` class to the roo
...
 
-o-tabs styles are also available via mixins. The base styles and buttontabs styles are available using the `oTabs` and `oTabsButtonTabs` mixins.
+```scss
+@include oTabs($opts: (
+    'themes': ('primary')
+));
+```
 
-If you're using the Sass mixins, you can also theme o-tabs using the `oTabsButtonTabsTheme` mixin, which can be set to any of the [`o-buttons` themes](https://github.com/Financial-Times/o-buttons#quick-start) by passing the name as a parameter:
+#### Mixin: `oTabsButtonTabsAddTheme`
```
Comment (notlee) [inline README.md]: Does `oTabsButtonTabsAddTheme` need to be public if I can:
```
@include oTabs($opts: (
    'themes': ('primary', 'mono')
));
```

Comment (rowanmanning) [reply README.md]: Yep because `mono` isn't included by default, and the registry uses mono

Comment (notlee) [reply README.md]: But the above snippet would work? As a Sass user I can include non-default themes without `oTabsButtonTabsAddTheme`? Should `oComponentAdd` be saved for custom things?

Comment (rowanmanning) [reply README.md]: Wait I have a suggestion, easier to commit it and I can roll back if necessary

Comment (rowanmanning) [reply README.md]: See https://github.com/Financial-Times/o-tabs/pull/80/commits/4023e81efc2f5dfad1ed7c0d6cd0b31a53c90286

File: README.md [hunk 4]
```diff
@@ -144,29 +141,59 @@ To apply the __buttontabs__ styling, add a `o-tabs--buttontabs` class to the roo
...
+    /* styles */
+}
+.o-tabs--primary {
+    /* styles */
+}
+/* etc. */
 ```
 
-Custom themes are also supported using `oTabsButtonTabsTheme` and the same custom theme map as for [`o-buttons` custom themes](https://github.com/Financial-Times/o-buttons#custom-themes).
+If you wish to specify a subset of themes to output styles for, you can pass in options (see [themes](#themes) for available options):
```
Comment (notlee) [inline README.md]:
```suggestion
If you wish to specify more themes to output styles for, you can pass in options (see [themes](#themes) for available options):
```

**Pull request merged**